openapi.yaml

```diff
@@ -6279,13 +6261,19 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/TemplateSharing'
+              type: array
```

Comment:
I could this change is related to bulk sharing.

Related UI change moved to staging without SDK update ? can you confirm @deekshith1856

Reply:
Yes, the change is moved to staging without updating the sdk, but its working in staging without any issues. So this update will resolve that issue without requiring any further UI changes